.github/workflows/deploy.yml

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
Why are you adding this entire job matrix? It seems quite excessive and most of it (if not all) should be covered by the E2E

Reply:
It’s covered in Shipyard, but not here. There are _no_ tests of the bundle in the operator, which is unfortunate since that’s where it gets changed. Catching failures in Shipyard doesn’t prevent merging the PR causing them in the operator.